features/gba-dev/fields-mapping-gba.csv

```diff
@@ -8,169 +8,71 @@ adressering.adresregel1,verblijfplaats.straat|verblijfplaats.huisnummer|verblijf
 adressering.adresregel2,gemeenteVanInschrijving|verblijfplaats.postcode|verblijfplaats.woonplaats|verblijfplaats.regel2
 adressering.adresregel3,verblijfplaats.regel3
 adressering.gebruikInLopendeTekst,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
-adressering.inOnderzoek,persoonInOnderzoek|verblijfplaats.inOnderzoek|partners.inOnderzoek
-adressering.inOnderzoek.aanhef,persoonInOnderzoek|partners.inOnderzoek
-adressering.inOnderzoek.aanschrijfwijze,persoonInOnderzoek|partners.inOnderzoek
-adressering.inOnderzoek.adresregel1,verblijfplaats.inOnderzoek
-adressering.inOnderzoek.adresregel2,verblijfplaats.inOnderzoek
-adressering.inOnderzoek.adresregel3,verblijfplaats.inOnderzoek
-adressering.inOnderzoek.datumIngangOnderzoekPartner,partners.inOnderzoek
-adressering.inOnderzoek.datumIngangOnderzoekPersoon,persoonInOnderzoek
-adressering.inOnderzoek.datumIngangOnderzoekVerblijfplaats,verblijfplaats.inOnderzoek
-adressering.inOnderzoek.gebruikInLopendeTekst,persoonInOnderzoek|partners.inOnderzoek
-adressering.inOnderzoek.land,verblijfplaats.inOnderzoek
 adressering.land,verblijfplaats.land
 adressering.land.code,verblijfplaats.land.code
 adressering.land.omschrijving,verblijfplaats.land.omschrijving
-geboorte.inOnderzoek,persoonInOnderzoek
-geboorte.inOnderzoek.datum,persoonInOnderzoek
-geboorte.inOnderzoek.datumIngangOnderzoek,persoonInOnderzoek.datumIngangOnderzoek
-geboorte.inOnderzoek.land,persoonInOnderzoek
-geboorte.inOnderzoek.plaats,persoonInOnderzoek
+adresseringBinnenland,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum|gemeenteVanInschrijving|verblijfplaats.straat|verblijfplaats.huisnummer|verblijfplaats.huisletter|verblijfplaats.huisnummertoevoeging|verblijfplaats.aanduidingBijHuisnummer|verblijfplaats.postcode|verblijfplaats.woonplaats|verblijfplaats.locatiebeschrijving|verblijfplaats.land|verblijfplaats.regel1|verblijfplaats.regel2|verblijfplaats.regel3
+adresseringBinnenland.aanhef,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
+adresseringBinnenland.aanschrijfwijze,naam|geslacht|partners.naam.adellijkeTitelPredicaat|partners.naam.voorvoegsel|partners.naam.geslachtsnaam|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
+adresseringBinnenland.aanschrijfwijze.aanspreekvorm,naam.adellijkeTitelPredicaat|geslacht|partners.naam.adellijkeTitelPredicaat|partners.aangaanHuwelijkPartnerschap.datum|partners.ontbindingHuwelijkPartnerschap.datum
```

Comment:
Moet je hiervoor niet ook 'aanduidingNaamgebruik' en 'geslacht' terug krijgen van de GBA API?

Reply:
geslacht staat er al bij, toch?

naam.aanduidingNaamgebruik ontbrak inderdaad. voeg ik toe

Reply:
Is correct nu.